src/components/Filters/ActualSwitch.tsx

```diff
@@ -3,8 +3,18 @@ import { Row, Switch } from 'antd';
 import TooltipHandler from './TooltipHandler';
 import { styles } from '../../utils/constants';
 
-class ActualSwitch extends Component {
-    constructor(props) {
+interface Props {
+    //TODO: should an interface for this inner object be created?
```

Comment:
@BenjiLee should an interface for this inner object be created?

Reply:
I think so. You will want to define something more than just an `object`.

Reply:
Looks like the object looks like this https://github.com/HopkinsIDD/covid-dashboard-app/blob/3b3a194abfa1a0905f666b750f0708235af5c5f4/src/utils/utils.js#L77

I would define an interface here and leave a comment to use this interface in the utils file. We can't migrate the utils file yet because it touches too many non TS files right now.